src/index.js

```diff
@@ -16,11 +17,15 @@ if (process.env.NODE_ENV !== "production") {
 
 const mapConfig = parseUriHash(defaultMapConfig);
 
+new InfoWindow();
+new TitleDetails();
+new LoadingIndicator();
+
 i18nInit().then(() => {
-  new LeafletMap(mapConfig);
+  Window.lmap = new LeafletMap(mapConfig);
```

Comment:
@jimboweb Again, you shouldn’t need to add this to the window if you made the class method use an arrow function that’s called using dispatch